main.js

```diff
@@ -170,3 +170,52 @@ function validationForm(event) {
 submitBtn.addEventListener('click', validationForm);
 
 // -------Start Section Form Validations-----------------//
+
+// -------Start Local Storage Section ------------------//
+
+// Const
+
+const formName = contactForm.querySelector('#name');
+const formEmail = contactForm.querySelector('#email-input');
+const formMsg = contactForm.querySelector('#user-message');
+
+// create an object to save data from form;
+const formData = {
+  name: '',
+  email: '',
+  message: '',
+};
+// first we check if available localStorage, need to create a function to do so
+
+function displayFromLocalStorage() {
+  document.querySelector('#name').value = formData.name;
+  document.querySelector('#email-input').value = formData.email;
+  document.querySelector('#user-message').value = formData.message;
+}
+
+function getLocalStorage() {
+  const currentData = localStorage.getItem('formData');
+  const getFormData = JSON.parse(currentData);
+  formData.name = getFormData.name;
+  formData.email = getFormData.email;
+  formData.message = getFormData.message;
+  displayFromLocalStorage();
+}
```

Comment:
- [ ] You did great implementing the local storage data preserve :+1: :100: but I think there are still requirements you need to meet before moving to the next milestone. 
> When the user loads the page, if there is any data in the local storage the input fields are pre-filled with this data.
- - Looking at the project requirement, when the page loads you are meant to check if the user has any previously stored data, then get it from the local storage and populate it in the input field, you currently haven't implemented this, I'll suggest you go through your code base and implement this feature. :smile: :small_airplane: :wave: